jupyterhub/tests/mocking.py

```diff
@@ -200,17 +200,20 @@ def init_signal(self):
 
     def load_config_file(self, *args, **kwargs):
         pass
-    
+
     @gen.coroutine
     def initialize(self, argv=None):
-        self.db_file = NamedTemporaryFile()
         self.pid_file = NamedTemporaryFile(delete=False).name
-        self.db_url = self.db_file.name
+        self.db_file = NamedTemporaryFile()
+        self.db_url = os.getenv('JUPYTERHUB_TEST_DB_URL') or self.db_file.name
```

Comment:
Assuming the env variable will let us test a basic Postgres and mysql db in Travis going forward.